BraintreeCore/src/main/java/com/braintreepayments/api/PaymentMethodClient.java

```diff
@@ -26,18 +31,46 @@ public PaymentMethodClient(BraintreeClient braintreeClient) {
         this.braintreeClient = braintreeClient;
     }
 
+    /**
+     * Parses a response from the Braintree gateway for a list of payment method nonces.
+     *
+     * @param jsonBody Json-formatted String containing a list of {@link PaymentMethodNonce}s
+     * @return List of {@link PaymentMethodNonce}s contained in jsonBody
+     * @throws JSONException if parsing fails
+     */
+    private static List<PaymentMethodNonce> parsePaymentMethodNonces(String jsonBody) throws JSONException {
+        JSONArray paymentMethods =
+            new JSONObject(jsonBody).getJSONArray(PAYMENT_METHOD_NONCE_COLLECTION_KEY);
+
+        if (paymentMethods == null) {
+            return Collections.emptyList();
+        }
+
+        List<PaymentMethodNonce> paymentMethodNonces = new ArrayList<>();
+        JSONObject json;
+        PaymentMethodNonce paymentMethodNonce;
+        for(int i = 0; i < paymentMethods.length(); i++) {
+            json = paymentMethods.getJSONObject(i);
+            // TODO: leverage compileOnly gradle dependencies to return strongly-typed nonces (e.g. CardNonce, PayPalAccountNonce etc.)
+            paymentMethodNonce = PaymentMethodNonce.fromJSON(json);
+            if (paymentMethodNonce.getType() != PaymentMethodType.GOOGLE_PAY) {
+                paymentMethodNonces.add(paymentMethodNonce);
+            }
+        }
+
+        return paymentMethodNonces;
+    }
+
     /**
      * Retrieves the current list of {@link PaymentMethodNonce}s for the current customer.
      * <p>
      * When finished, the {@link java.util.List} of {@link PaymentMethodNonce}s will be sent to {@link
      * GetPaymentMethodNoncesCallback}
-     *
-     * @param context Android Context
-     * @param defaultFirst when {@code true} the customer's default payment method will be first in the list, otherwise
+     *  @param defaultFirst when {@code true} the customer's default payment method will be first in the list, otherwise
      *        payment methods will be ordered my most recently used.
```

Comment:
```suggestion
     *        payment methods will be ordered by most recently added.
```

Not related to this PR, but this method actually returns the most recently _added_ payment methods, not the most recently used. We had [an issue about this](https://github.com/braintree/braintree-ios-drop-in/issues/180) on iOS Drop-in.